### ISSUE TITLE
The learning rate for different part of parameters

### ISSUE THREAD

**Issue by YangWangsky** [hellochick/PSPNet-tensorflow#46]
It's a nice work, thanks for the code provided by the owner. And I notice the following the comment:
# According from the prototxt in Caffe implement, learning rate must multiply by 10.0 in pyramid module.
I have found the Caffe implement by Hao Zhang, however I can't find the prototxt for training. How can I get it?
 
Thanks.

**Comment by hellochick** [comment]
@YangWangsky ,

The original author didn't provide the training code. So I tried to implement followed by the paper description.

**Comment by YangWangsky** [comment]
@hellochick 
Thanks for your reply. I also read the original paper, and there seems no description about different learning rate for the different parts. Could you please help me find the setting for learning rate?

**Comment by hellochick** [comment]
Hey,

If you take a look at the details in the [original caffe prototxt](https://github.com/hszhao/PSPNet/blob/master/evaluation/prototxt/pspnet101_cityscapes_713.prototxt), you can find these settings.

For example, look at line 7062-7070, you can see these codes:
```
layer {
  name: "conv5_4/bn"
  type: "BN"
  bottom: "conv5_4"
  top: "conv5_4"
  param {
    lr_mult: 10
    decay_mult: 0
  }
``` 
And this indicates your learning rate multiplied by 10.

**Comment by YangWangsky** [comment]
@hellochick 
Ok, I get it. I have not used caffe before, thank you for your reply！！
